### ISSUE TITLE
Warning semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown

### ISSUE THREAD

**Issue by yichuan9527** [JunYeopLee/fast-autoaugment-efficientnet-pytorch#1]
Hello，I run `python train.py --seed=24 --scale=3 --optimizer=sgd --fast_auto_augment=True
`. A warning is reported as follow:
`/disk1/jinlin2/anaconda3/envs/fast-auto-aug/lib/python3.7/multiprocessing/semaphore_tracker.py:144: UserWarning: semaphore_tracker: There appear to be 1 leaked semaphores to clean up at shutdown
  len(cache))`

**Comment by JunYeopLee** [comment]
Hi,

I'm sorry for the late comment. Now I see you left an issue.

I tested the project only with python 3.6. Can you try with the lower version of python?
